cpp/src/aztec/dsl/acir_format/ecdsa_secp256k1.cpp

```diff
@@ -103,7 +103,18 @@ void create_ecdsa_verify_constraints(plonk::TurboComposer& composer, const Ecdsa
     stdlib::ecdsa::signature<plonk::TurboComposer> sig{ stdlib::byte_array<plonk::TurboComposer>(&composer, rr),
                                                         stdlib::byte_array<plonk::TurboComposer>(&composer, ss) };
 
-    auto pub_key = secp256k1_ct::g1_ct(pub_key_x_fq, pub_key_y_fq);
+    pub_key_x_fq.assert_is_in_field();
+    pub_key_y_fq.assert_is_in_field();
+
+    // TODO: crypto-dev to fix calculation and constraining of the signature result is correct
```

Comment:
Since this has nothing to do with with the PR title -- I think this should be brought up as an issue on the original PR and we can resolve it there

Reply:
@kevaundray this PR is targeting my branch, so we can merge it in now and get it resolved as part of #198

Reply:
Ok will bring up as an issue. 
Originally these comments were just this line:
`auto pub_key = secp256k1_ct::g1_ct(pub_key_x_fq, pub_key_y_fq);` 
Which was giving unused variable errors, so I did these changes for now.